### PR TITLE
Added eslint and formatted package

### PR DESCRIPTION
**Pull request by divmgl**
Merge after https://github.com/koajs/koa-router/pull/1 @dead-horse

### CODE REVIEW

**Comment by dead-horse** [comment]
I'd like to keep the style as original for now, people can migrate PR's from koa-router much easier

**Comment by divmgl** [comment]
Okay I'll make the changes

**Comment by divmgl** [comment]
@dead-horse there was very little consistency in the codebase so it's going to be tricky to create the perfect set of eslint rules. For now, I've added back semi-colons so the pull request should be much more readable. Let me know your thoughts.

**Comment by divmgl** [comment]
Can we get this merged so all pull requests going forward have a formatting style? @dead-horse

**Comment by divmgl** [comment]
@dead-horse ?

File: .gitignore [hunk 1]
```diff
@@ -30,7 +30,6 @@ build/Release
 node_modules
 jspm_packages
 
-package-lock.json
```
Comment (dead-horse) [inline .gitignore]: keep package-lock ignored, others LGTM

**Comment by niftylettuce** [comment]
Thanks for PR - I will be integrating prettier/xo setup from https://lass.js.org into this project.  If you're interested in this approach and get time before I do, you could practically clone my work/setup I have done in projects like https://github.com/cabinjs/parse-logs.

**Pull request closed without merging**